### ISSUE TITLE
Current user  and authentication

### ISSUE THREAD

**Issue by 8666** [zadam/trilium#934]
I can not see the username. When I installed it for the first time I entered a random username and a common pass. Later after installing the server and setting up the sync, it all worked but I still did not know the username :) to use the Web UI... so I started all over.

It would be nice to see the current username in the interface. Or even better asks for it just like on the Web UI. For some strange reasons user/pass is not required when starting the desktop app. It would be more secure to have a password prompt (the last username already filled)

I know that the multi-user setup is not in the plan... but running the server just for me is overkill. Why the server can not be used by multiple people with own user/pass. No need for sharing etc. Just individual databases.

**Comment by zadam** [comment]
You're right that not seeing username is annoying. I added its display into the Options -> Username & Password, will be in 0.41.

Desktop app does not require username & password since normal (not protected) notes are not encrypted and so if you have an access to the desktop app, you have the access to the data even without the app (by reading the sqlite database, does not apply to protected notes).

**Comment by zadam** [comment]
SInce 0.50 there's no "user" anymore.

**Comment by krescent** [comment]
> SInce 0.50 there's no "user" anymore.

Would like to ask inside the trilium web clipper, while I am trying to configure a server connection, it required me to input a username, what should I do to avoid it? Many thanks.

**Comment by zadam** [comment]
@krescent Just use "trilium" or whatever else as username, it's ignored. It's still in web clipper to support the older versions of trilium which required the username.
